lib/domains/character/types.tsx

```diff
@@ -182,14 +184,37 @@ export type IImageBlock = {
   value: string;
 };
 
+export type ILinkBlock = {
+  type: BlockType.Link;
+  meta: {
+    helperText?: string;
+    displayName?: string;
+    editName?: boolean;
+  };
+  value: {
+    link: string;
+    isValid: boolean;
```

Comment:
**suggestion** I don't think we need a `isValid` option saved in the character sheet. Perhaps we could keep the value as a simple `string` and move the isValid as a dynamic const calculated by the Link block component. What do you think ?

Reply:
I like it :) Will do